packages/manager/modules/pci/src/projects/project/project.service.js

```diff
@@ -0,0 +1,14 @@
+export default class {
+  /* @ngInject */
+  constructor() {
+    this.project = {};
+  }
+
+  getProjectInfo() {
+    return this.project;
+  }
+
+  setProjectInfo(project = {}) {
```

Comment:
Statefull services is not recommended and should be avoided if possible (it makes code really harder to debug since you never know the service current state and different part of the code can alter the service behavior).

What are you trying to achieve, am i missing something here ? maybe you can pass the project to your component as a binding ?

Reply:
The project information is required increase-request controller which is 2 levels below the project controller (which has required info) project -> quota -> increase-request.
And moreover the quota component is loaded from sidebar-component. All the siblings of quota are receiving only projectId in a generalised manner. So adding other info will be difficult. that's the reason i did not go ahead with passing data using component binding approach.
Now, the other options left were,
1) to make a separate HTTP call in increase-request controller to fetch the required info 
2) maintain the info in service to avoid an extra call.